roles/repo_setup/tasks/ci_mirror.yml

```diff
@@ -8,10 +8,24 @@
   become: "{{ not cifmw_repo_setup_output.startswith(ansible_user_dir) }}"
   when:
     - mirror_path.stat.exists
-  ansible.builtin.shell: |
-    set -o pipefail
-    source /etc/ci/mirror_info.sh
-    sed -i -e "s|https://trunk.rdoproject.org|$NODEPOOL_RDO_PROXY|g" *.repo
-    sed -i -e "s|http://mirror.stream.centos.org|$NODEPOOL_CENTOS_MIRROR|g" *.repo
-  args:
-    chdir: "{{ cifmw_repo_setup_output }}"
+  block:
+    - name: Use RDO proxy mirrors
+      ansible.builtin.shell: |
+        set -o pipefail
+        source /etc/ci/mirror_info.sh
+        sed -i -e "s|https://trunk.rdoproject.org|$NODEPOOL_RDO_PROXY|g" *.repo
+      args:
+        chdir: "{{ cifmw_repo_setup_output }}"
+
+    # TODO(rlandy) remove when CentOS 10 mirrors exist on Nodepool Hosts
```

Comment:
Thank you for the additional explanation. Please add this note to the `name`. It will be visible every time during the test run, so we won't loose track of it.